scaffolding-ruby/lib/scaffolding.sh

```diff
@@ -771,6 +772,17 @@ _detect_rails5_app() {
     return 1
   fi
 }
+_detect_rails6_app() {
+  if _has_gem railties && _compare_gem railties \
+      --greater-than-eq 6.0.0 --less-than 6.1.0; then
```

Comment:
Would it make sense to make this `--less-than 7.0.0`?   Rails appears to fairly consistent with their versions, and would give us some room if they release a 6.2 or greater version of Rails.

Reply:
Rails' versioning is not quite the Semantic Versioning you would expect. It uses a "[shifted version](https://guides.rubyonrails.org/maintenance_policy.html) of [semver](https://semver.org/)." Rails maintenance policy warns that changes in what we would call the minor version (the `Y` in `X.Y.Z`) means, "[n]ew features, may contain API changes (Serve as major versions of Semver)." So, yea, there can be surprising changes going from Rails 6.0 to 6.1.